src/firewall.py

```diff
@@ -159,20 +149,14 @@ def firewall_disallow(
     elif upnp_only:
         ipvs = []
 
-    changed = False
-
     for p in protocols:
         # Iterate over IP versions to remove port
         for i in ipvs:
             if port in firewall[i][p]:
                 firewall[i][p].remove(port)
-                changed = True
             else:
                 ipv = "IPv%s" % i[3]
-                if not reload_only_if_change:
-                    logger.warning(
-                        m18n.n("port_already_closed", port=port, ip_version=ipv)
-                    )
+                logger.warning(m18n.n("port_already_closed", port=port, ip_version=ipv))
```

Comment:
Ugh actually we probably want to have an option to disable this warning, otherwise everytime a port is not to be "exposed" (= the vast majority of the apps ?), yunohost will display a warning during install/upgrade/... when provision ports

Similar stuff for `port_already_open` too